chsdi/models/vector/uvek.py

```diff
@@ -807,6 +807,47 @@ class StatistikwasserkraftanlagenNew(Base, Vector):
 register('ch.bfe.statistik-wasserkraftanlagen', StatistikwasserkraftanlagenNew)
 
 
+class Erneuerbarheizen(Base, Vector):
+    __tablename__ = 'erneuerbarheizen'
+    __table_args__ = ({'schema': 'bfe', 'autoload': False})
+    __template__ = 'templates/htmlpopup/erneuerbarheizen.mako'
+    __bodId__ = 'ch.bfe.erneuerbarheizen'
+    __queryable_attributes__ = ['company']
+    __label__ = 'company'
+    id = Column('bgdi_id', Integer, primary_key=True)
```

Comment:
i would use xtf_id as id / primary key in the model and also create an pk constraint / Index in the table.

Reply:
Make sense, i do the change